packages/aila/src/lib/agentic-system/agents/sectionAgents/starterQuizAgent/starterQuiz.instructions.ts

```diff
@@ -0,0 +1,18 @@
+import { quizQuestionDesignInstructions } from "../shared/quizQuestionDesign.instructions";
+
+export const starterQuizInstructions = `# Task
+
+Create a 6-question multiple-choice quiz to assess PRIOR KNOWLEDGE ONLY — do not include or hint at new lesson content.
```

Comment:
Resulting from our call: if the agent can't see the current prior knowledge field then this will probably be a regression in performance